### PR TITLE
Initial functionality

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR enables the creation of a security auditor group, which includes the default AWS `SecurityAudit` policy, as well as a policy containing some additional permissions.

## 💭 Motivation and context ##

This repo was created because we received a request to define security auditors for our CyHy AWS account.  Note that this repo is very similar to [`cisagov/cool-auditor-iam`](https://github.com/cisagov/cool-auditor-iam), except that it does not require a financial auditor group and it only applies to a single AWS account, rather than a group of accounts in the same AWS organization.

## 🧪 Testing ##

I successfully applied these changes in the CyHy production account and confirmed that the new security auditor group was properly created and that it contained the users that I specified.

~⚠️ Note that automated tests will fail until https://github.com/cisagov/skeleton-tf-module/pull/186 is merged and its changes trickle down to this repository.~

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by jasonodoom — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]
_Approval intensifies!!!_

File: variables.tf [hunk 1]
```diff
@@ -4,30 +4,43 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)."
+variable "auditors" {
+  type        = map(map(list(string)))
```
Comment (mcdonnnj) [inline variables.tf]: Just as a thought but would this be better defined as:

```suggestion
  type        = map(object({ roles = list(string) }))
```

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪💪💪 I have one comment for your consideration but otherwise this PR looks short and sweet.

File: variables.tf (hunk 1 above)
Comment (dav3r) [reply variables.tf]: You are correct!  Please see https://github.com/cisagov/cyhy-auditor-iam/pull/3/commits/0be4c7ac003bd92310b7e2b612b4e9f8747d7f13.

**Pull request merged**